### DISCUSSION TITLE
개발문서를 공유할 좋은 플랫폼

### DISCUSSION THREAD

**Discussion by Cotidie** [ApptiveDev/.github#5] [Q&A] [answered]
앞으로 더 많은 개발문서를 작성하게 될텐데
공동편집, 공유, 검색 등이 편리한 플랫폼이 어디 없을까요?

Confluence가 완벽하지만 10명 이상은 유료이고
Notion은 소유자가 1명이라 운영진이 바뀌면 계정도 바꿔야 하고요.

**Comment by Kick-snare** [comment, marked as answer]
아무래도 github에서 markdown으로 쓰고 git으로 관리하는게 가장 편하지 않을까요? 예를 들면 기존의 [여기](https://github.com/ApptiveDev/.github) 라던지... notion, confluence 등등 뭐든 제 3의 또다른 플랫폼을 거쳐야 한다면 접근성이 떨어질 것 같습니다. 

아니면 github안에 [위키](https://github.com/ApptiveDev/.github/wiki)를 활용하는 방법도 있을거 같네요

**Reply by Cotidie** [reply]
좋네요, .github 리포의 위키를 먼저 사용해볼게요

문서가 많아지거나, 검색하는 면에서 조금 걱정이었는데 리포지토리를 분리하면 될 것 같군요.
다만 깃헙 무료버전은 공개 리포만 위키가 가능한 탓에 접근에 제한이 없어서 내부 문서는 따로 관리해야겠습니다.

**Comment by Cotidie** [comment]
다음과 같은 문서를 써보려고 합니다

- [x] (백엔드) 리눅스 중요성 및 설치, 유용한 플러그인 (zsh, auto-suggestion, tmux,  KDE plasma)
- (백엔드) 서비스 규모에 따른 스케일 관리 로드맵
- (백엔드) 서비스 주요 지표 모니터링 (Prometheus, Grafana, Opentelemetry)
- (백엔드) 한 VM에서 여러 서비스 호스팅하기 (Docker, Nginx)
- (백엔드) 실시간 데이터 및 REST API 동시에 제공하기 (REST, gRPC, Web Socket)
- [x] (공통) 동아리 리포지토리 설정 방법 (Squash Merge, Branch Protection, Branch Auto Deletion, Code Review Approval, JIRA 연동)
- (공통) 동아리 개발 협업 사이클 (JIRA/Github Team 활용법, 티켓 및 branch 관리) 
- (공통) 애자일 개발 사이클
- (공통) 커밋 내역 포함 리포지토리 복제 방법 (git, copybara)
- (공통) 프로젝트 README 작성 가이드라인
- (공통) 동아리 필수 CI
- [x] (공통) git 스터디/워크샵
